docs/app-integration-development/20-OAuth-2-Extended-OAuth.md

```diff
@@ -0,0 +1,67 @@
+---
+tags: [app-integration-development]
+---
```

Comment:
Do you plan to add a list of API endpoints supported by API Scopes, or is that something we would depend on the respective product teams to update their documentation and we just add a blurb that endpoints supporting API Scopes will have the appropriate tag/notice?